assets/aml-benchmark/tests/dataset_downloader/test_dataset_downloader.py

```diff
@@ -175,7 +177,7 @@ def test_invalid_hf_dataset(
         if dataset_name == "winogrande" and configuration is None:
             expected_exception_mssg = (
                 f"Multiple configurations available for dataset '{dataset_name}'. Please specify either one of "
-                f"the following: {get_dataset_config_names(dataset_name)} or 'all'."
+                f"the following: {get_dataset_config_names(dataset_name, trust_remote_code=True)} or 'all'."
```

Comment:
what remote_code are we anticipating to execute here?

Reply:
This issue might have surfaced up because of the updated datasets library version. 

![image](https://github.com/user-attachments/assets/f47af709-88e1-4253-b84f-7e02dff6ec87)


More details
https://huggingface.co/docs/datasets/main/en/load_hub#remote-code